lambdas/accesstoken/src/test/java/uk/gov/di/ipv/cri/passport/accesstoken/validation/TokenRequestValidatorTest.java

```diff
@@ -45,14 +48,17 @@ class TokenRequestValidatorTest {
 
     private TokenRequestValidator validator;
     @Mock private ConfigurationService mockConfigurationService;
+    @Mock private ClientAuthJwtIdService mockClientAuthJwtIdService;
```

Comment:
```suggestion
    @Mock private ConfigurationService mockConfigurationService;
    @Mock private ClientAuthJwtIdService mockClientAuthJwtIdService;
    @InjectMocks private TokenRequestValidator validator;
```

This way you can drop the instantiation in the setup method

Reply:
Having some trouble with this one as it also needs the `verifier` injecting, but seems we are exercising the real implementation of `verifier` in the tests so I don't think we want it mocked?

Reply:
Oh interesting, I missed the fact that we're creating the verifier in the constructor.